docs/av-stack-reorder.1.md

```diff
@@ -0,0 +1,27 @@
+# av-stack-reorder
+
+## NAME
+
+av-stack-reorder - Interactively reorder the stack
+
+## SYNOPSIS
+
+```synopsis
+av stack reorder [--continue | --abort]
+```
+
+## DESCRIPTION
+
+This is analogous to git rebase --interactive but operates across all branches
+in the stack.
+
+Branches can be re-arranged within the stack and commits can be edited,
+squashed, dropped, or moved within the stack.
```

Comment:
Might be worth explicitly saying that the commits can be moved across the branches.

Reply:
Done.